web/tsconfig.base.json

```diff
@@ -48,6 +48,13 @@
                     "no-unknown-property": "off",
                     "no-unknown-attribute": "off"
                 }
+            },
+            {
+                "name": "@genesiscommunitysuccess/custom-elements-lsp",
+                "designSystemPrefix": "ak-",
```

Comment:
This enables TSC and TSServer to identify when web components are not being invoked correctly.  I've limited the CEM check to only our own components.